src/ostorlab/cli/scan/run/assets/ios_testflight.py

```diff
@@ -0,0 +1,27 @@
+"""This module triggers a scan using the ios_testflight asset."""
+
+import logging
+
+import click
+
+from ostorlab.cli.scan.run import run
+from ostorlab.cli import console as cli_console
+from ostorlab.assets import ios_testflight as ios_testflight_asset
+
+console = cli_console.Console()
+logger = logging.getLogger(__name__)
+
+
+@run.run.command(name="ios-testflight")
+@click.option("--application-url", required=False)
+@click.pass_context
+def ios_store(ctx: click.core.Context, application_url: str) -> None:
+    """Run scan using testflight url."""
+    runtime = ctx.obj["runtime"]
+    assets = [ios_testflight_asset.IOSTestflight(application_url=application_url)]
```

Comment:
you need to check if application_url is None since IOSTestflight accept str

Reply:
It is a mandatory field so it cannot be called with a None value